dev-vcs/easygit/easygit-1.7.5.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="eg"
```

Comment:
Please remove this variable. It doesn't really help with anything, and replaces two characters with eight…